### PR TITLE
build: update ipyvue to 1.7

### PR DESCRIPTION
**Pull request by 12rambau**
In https://github.com/widgetti/ipyvue/pull/40 a class management tool have been introduced in ipyvue and this functionality is documented in the latest version of ipyvuetify. to make sure it's available, ipyvue should be at least 1.7

### CODE REVIEW

**Comment by maartenbreddels** [comment]
I think this is something that downstream users/libraries should do since ipyvuetify doesn't depend on it right?

**Comment by 12rambau** [comment]
As it's already documented here: https://ipyvuetify.readthedocs.io/en/latest/usage.html#classlist-attribute,
I thought it would make sense to ensure it works (I experienced it in a messed up env. and that's a pain to diagnose). Or there should be a disclaimer in the doc saying: "you need to make sure ipyvue>1.7 to use this functionality" but it looks weird to me as ipyvuetify is relying on ipyvue to generate the widgets.

**Comment by maartenbreddels** [comment]
Good point, what do you think Mario?

File: setup.py
Comment (12rambau) [inline setup.py:181]:
```suggestion
    install_requires = [
```

**Comment by 12rambau** [comment]
@mariobuikhuizen should I change anything ?

**Review by maartenbreddels — APPROVED** [review]

**Comment by maartenbreddels** [comment]
can you make rebase/squash this?

**Comment by 12rambau** [comment]
@maartenbreddels done. Note that for all the PR I propose, I assume that you'll use the "squash and merge" so I authorize myself lots of noise

**Comment by maartenbreddels** [comment]
True, and it's not an issue, actually, my mistake. Only after selecting 'squash and merge' the merge button was green, never saw that before.

**Pull request merged**
